quality_of_service_demo/src/common_nodes.cpp

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <chrono>
+#include <string>
+
+#include "quality_of_service_demo/common_nodes.hpp"
+
+using namespace std::chrono_literals;
+
+Talker::Talker(
+  const std::string & topic_name,
+  rclcpp::PublisherOptions<> pub_options,
```

Comment:
I think the `<>` are no longer required due to this: https://github.com/ros2/rclcpp/blob/d34fa607a2061372d77f2e3d59340bb6eae7fe63/rclcpp/include/rclcpp/publisher_options.hpp#L40